paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/MultiTablesCompactorSink.java

```diff
@@ -121,7 +122,9 @@ protected DataStreamSink<?> doCommit(
         if (streamingCheckpointEnabled) {
             assertStreamingConfiguration(env);
         }
-
+        if (!options.get(SINK_COMMITTER_OPERATOR_CHAINING)) {
```

Comment:
It is better to pass this bool to `CommitterOperator`. It can set `setChainingStrategy` to `ChainingStrategy.NEVER`.